src/less/mixins/typography/base/typography-mixins.less

```diff
@@ -29,9 +29,9 @@
 }
 
 .typography-large-1() {
-    font-size: @font-size-large-1;
+    font-size: @font-size-large-2;
```

Comment:
Can you double check these 4 typography code changes? Not sure it's intended or correct...

![Screen Shot 2021-11-17 at 2 56 39 PM](https://user-images.githubusercontent.com/38065/142295864-dd6af293-2e8e-4991-8ef3-7321adf7171b.png)